src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -253,6 +263,231 @@ static bool CollectCrashDump(Process process, string path)
+
+                    if (isNative)
+                    {
+                        string nativeModuleName = (string)frame["native_module"];
+                        string unmanagedName = (string)frame["unmanaged_name"];
+
+                        if ((nativeModuleName != null) && (knownNativeModules.Contains(nativeModuleName)))
+                        {
+                            // Need to use llvm-symbolizer (only if module_address != 0)
+                            AppendAddress(addrBuilder, coreRoot, nativeModuleName, (string)frame["native_address"], (string)frame["module_address"]);
+                        }
+                        else if ((nativeModuleName != null) || (unmanagedName != null))
+                        {
+                            if (nativeModuleName != null)
+                            {
+                                addrBuilder.Append($"{nativeModuleName}!");
+                            }
+                            if (unmanagedName != null)
+                            {
+                                addrBuilder.Append($"{unmanagedName}");
+                            }
+                        }
+                    }
+                    else
+                    {
+                        string fileName = (string)frame["filename"];
+                        string methodName = (string)frame["method_name"];
+
+                        if ((fileName != null) || (methodName != null))
+                        {
+                            // found the managed method name
+                            if (fileName != null)
+                            {
+                                addrBuilder.Append($"{fileName}!");
+                            }
+                            if (methodName != null)
+                            {
+                                addrBuilder.Append($"{methodName}");
+                            }
+                        }
+                        else
+                        {
+                            addrBuilder.Append($"{frame["native_address"]}");
+                        }
+                    }
+                    addrBuilder.AppendLine();
+
+                }
+            }
+
+            string symbolizerOutput = null;
+
+            Process llvmSymbolizer = new Process()
+            {
+                StartInfo = {
+                    FileName = "llvm-symbolizer",
+                    Arguments = $"--pretty-print",
+                    UseShellExecute = false,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    RedirectStandardInput = true,
+                }
+            };
+
+            outputWriter.WriteLine($"Invoking {llvmSymbolizer.StartInfo.FileName} {llvmSymbolizer.StartInfo.Arguments}");
+            if(!llvmSymbolizer.Start())
+            {
+                outputWriter.WriteLine($"Unable to start {llvmSymbolizer.StartInfo.FileName}");
```

Comment:
This can throw an exception if the symbolizer is not present - all this should be best effort and wrapped in try-catch.

Reply:
I ran this code all this time when `llvm-symbolizer` was not present and it worked fine. I will still wrap it up in try-catch, just to be sure.